### PR TITLE
packages\react-devtools-shared\src\hook.js  --> fixed typos

### PR DESCRIPTION
**Pull request by Biki-das**
evaling --> evading
envified --> envied
eval'ed --> evaded

### CODE REVIEW

File: packages/react-devtools-shared/src/hook.js [hunk 1]
```diff
@@ -334,7 +334,7 @@ export function installHook(target: any): DevToolsHook | null {
     // For React Native, we intentionally patch early (during injection).
     // This provides React Native developers with components stacks even if they don't run DevTools.
     //
-    // This won't work for DOM though, since this entire file is eval'ed and inserted as a script tag.
+    // This won't work for DOM though, since this entire file is evaded and inserted as a script tag.
```
Comment (akgupta0777) [inline packages/react-devtools-shared/src/hook.js:337]: The word eval'ed means evaluated and it's not same as evaded. Other corrections are not typos actually.

**Review by akgupta0777 — COMMENTED** [review]
Not needed.

File: packages/react-devtools-shared/src/hook.js (hunk 1 above)
Comment (Biki-das) [reply packages/react-devtools-shared/src/hook.js:337]: may be evaled is evaluated but evaling has no meaning and evading is the closest to the actual meaning fitt to the sentence

Comment (Biki-das) [reply packages/react-devtools-shared/src/hook.js:337]: > Not needed.

Never reply "not needed" unless you are a part of the maintainers, let them comment what they find incorrect the typos i have tried to fix first of all are typos ,evaling does not have any meaning, envified has no meaning , if you know it is not needed then sugges whats needed and that what maintainer of the project will do in a review

**Pull request closed without merging**